regtest/docker-compose.yml

```diff
@@ -0,0 +1,653 @@
+x-services:
+  base-scripts: &base-scripts
+    image: ${SCRIPTS_IMAGE}
+    volumes:
+      - bitcoin-data:/root/.bitcoin
+      - elements-data:/root/.elements
+      - cln1-data:/root/.lightning-1
+      - cln2-data:/root/.lightning-2
+      - lnd1-data:/root/.lnd-1
+      - lnd2-data:/root/.lnd-2
+      - boltz-client-data:/root/.boltz-client
+      - boltz-data:/root/.boltz-backend
+
+  base-lnd: &base-lnd
+    image: ${LND_IMAGE}
+    platform: linux/amd64
+    restart: always
+    depends_on:
+      bitcoind-init:
+        condition: service_completed_successfully
+    command:
+      - --listen=0.0.0.0:9735
+      - --rpclisten=0.0.0.0:10009
+      - --restlisten=0.0.0.0:8081
+      - --bitcoin.active
+      - --bitcoin.regtest
+      - --bitcoin.node=bitcoind
+      - --bitcoind.rpchost=bitcoind
+      - --bitcoind.zmqpubrawtx=bitcoind:29000
+      - --bitcoind.zmqpubrawblock=bitcoind:29001
+      - --bitcoind.rpccookie=/root/.bitcoin/regtest/.cookie
+      - --noseedbackup
+      - --protocol.wumbo-channels
+    expose:
+      - 8081
+      - 9735
+    volumes_from:
+      - bitcoind:ro
+    healthcheck:
+      test:
+        [
+          "CMD-SHELL",
+          "lncli --network=regtest --rpcserver=$$HOSTNAME --network=regtest getinfo",
+        ]
+      interval: 1s
+      timeout: 1s
+      retries: 30
+      start_period: 0s
+
+  base-cln: &base-cln
+    image: ${CLN_IMAGE}
+    platform: linux/amd64
+    restart: always
+    depends_on:
+      bitcoind-init:
+        condition: service_completed_successfully
+    command:
+      - --large-channels
+      - --network
+      - regtest
+      - --bind-addr=0.0.0.0:9735
+      - --bitcoin-rpcconnect=bitcoind
+      - --bitcoin-rpcport=18443
+      - --grpc-host=0.0.0.0
+      - --grpc-port=9736
+      - --dev-bitcoind-poll=3
+      - --developer
+      - --dev-fast-gossip
+      - --dev-fast-reconnect
+      - --plugin=/root/hold
+      - --plugin=/root/clnurl
+      - --clnurl-host=0.0.0.0
+      - --ignore-fee-limits=false
+      - --min-emergency-msat=1000000
+    expose:
+      - 9735
+      - 9736
+      # LNURL
+      - 3000
+    volumes_from:
+      - bitcoind:ro
+    healthcheck:
+      test: ["CMD", "lightning-cli", "--network=regtest", "getinfo"]
+      timeout: 1s
+      retries: 20
+      interval: 1s
+      start_period: 0s
+
+services:
+  cleanup:
+    <<: *base-scripts
+    restart: no
```

Comment:
I got this error:
```
services.cleanup.restart must be a string
```
All `restart:` values needed to be wrapped in double quotes (`restart: "no"`)

Reply:
Same here, my compose didn't mind it. Applied your fix in [06e22a9](https://github.com/breez/breez-sdk-liquid/pull/707/commits/06e22a9abfc961e056783a35ddd1fae696502426).